src/Commands/CreatePostgresUserForTenants.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Commands;
+
+use Illuminate\Console\Command;
+use Stancl\Tenancy\Concerns\HasTenantOptions;
+use Stancl\Tenancy\Jobs\CreatePostgresUserForTenant;
+
+class CreatePostgresUserForTenants extends Command
+{
+    use HasTenantOptions;
+
+    protected $signature = 'tenants:create-postgres-user {--tenants=* : The tenant(s) to run the command for. Default: all}';
```

Comment:
Not sure about using singular/plural for user here since if this can be/is meant to be executed for multiple tenants, using `users` might make more sense.

Reply:
> because we intend to create one user for each of the tenants

Ah, that'd make sense.

> But we'll most likely end up creating multiple different users

What do you mean by this?

Reply:
I mean we create user "foo" for tenant "foo", user "bar" for tenant "bar" and so on, so `users` might be good too. I still like `user` more

Reply:
Right, but it's user per tenant, so singular makes sense.